.github/templates/macos_binary_build_workflow.yml.j2

```diff
@@ -22,6 +22,14 @@ name: !{{ build_environment }}
           echo "MAC_PACKAGE_WORK_DIR=${RUNNER_TEMP}" >> "${GITHUB_ENV}"
 {%- endmacro %}
 
+{%- macro setup_python(py_ver) -%}
+      - name: Setup Python
+        uses: actions/setup-python@v5
+        with:
+          python-version: "!{{ py_ver.strip('t') }}.!{{0 if py_ver != '3.10' else 4}}!{{ '-rc.2' if '3.14' in py_ver else '' }}"
+          freethreaded: !{{ "true" if py_ver.endswith('t') else "false" }}
```

Comment:
You will not need this if you provide `python-version: {X}.{Y}t`

Reply:
@XuehaiPan do you know how to do it for 3.14?

Reply:
For pre-release versions (alpha, beta, rc), you can use:

```yaml
python-version: '3.12'

python-version: '3.13'

python-version: '3.13t'

python-version: '3.14t'
```

Reply:
I've tried it with `allow-prerelease` and it does not seem to work, see https://github.com/malfet/deleteme/blob/baad50f638bb4831232386f6629d673b04465c6a/.github/workflows/install_torch.yml#L25 and https://github.com/malfet/deleteme/actions/runs/17530584425/job/49786975781#step:2:12